test/run-tests

```diff
@@ -529,6 +518,13 @@ def get_comment_commands(gh, owner, repo, pullnr):
     return commands
 
 
+def get_status_context(config_name, image_name, ansible_ver_str):
+    suffix = ""
+    if config_name.endswith("-staging"):
+        suffix = " (staging)"
```

Comment:
I believe the instance name should be configurable instead of having only one option as a special case. There will be other instances than production and staging - if one is developing a change in test-harness or an alternative to it, one typically wants to test it before even pushing it to staging. There are many examples of this already where some PRs have results from some special CI runs. We may also have two different development branches running in parallel at the same time, like the qemu one and openstack one.

Reply:
ok - how about this https://github.com/linux-system-roles/test-harness/pull/87/commits/5e29e183f524819b1311167deffc61968e33ea7e

Reply:
How about changing `config_name` to `config_variant`, set it to `""` for the production instance and to `"staging"` for the staging instance and just do:

```
if config_variant:
    suffix = f" ({config_variant})"
else:
    suffix = ""
```

Reply:
> How about changing `config_name` to `config_variant`, set it to `""` for the production instance and to `"staging"` for the staging instance and just do:
> 
> ```
> if config_variant:
>     suffix = f" ({config_variant})"
> else:
>     suffix = ""
> ```
> 
> This avoids any magic strings like `linux-system-roles-test` that have a special meaning.

The reason I chose `config_name` as the name of the parameter is because this exactly corresponds to `config["name"]`.  Are you suggesting that we change `config["name"]` to `config["variant"]`?

Reply:
> The reason I chose `config_name` as the name of the parameter is because this exactly corresponds to `config["name"]`. Are you suggesting that we change `config["name"]` to `config["variant"]`?

yes, this seems to make more sense with these changes.

Reply:
https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0